frontend/packages/dev-console/src/components/helm/HelmReleaseResources.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { MultiListPage } from '@console/internal/components/factory';
+import { K8sResourceKind, kindForReference } from '@console/internal/module/k8s';
+import { DeploymentModel, StatefulSetModel, PodModel, SecretModel } from '@console/internal/models';
+import HelmResourcesListComponent from './HelmResourcesListComponent';
+import { flattenResources, helmReleaseResourceKindFilter } from './helm-release-resources-utils';
+import { ServiceModel } from '../../../../knative-plugin/src/models';
+
+export interface HelmReleaseResourcesProps {
+  obj: K8sResourceKind;
+}
+
+const HelmReleaseResources: React.FC<HelmReleaseResourcesProps> = ({ obj: resourceDetails }) => {
+  const { namespace } = resourceDetails.metadata;
+  const helmReleaseName = _.get(resourceDetails, 'metadata.labels.name');
+  const resources = [
+    {
+      kind: DeploymentModel.kind,
+      namespaced: true,
+      isList: true,
+      selector: { release: `${helmReleaseName}` },
+    },
+    {
+      kind: ServiceModel.kind,
+      namespaced: true,
+      isList: true,
+      selector: { release: `${helmReleaseName}` },
+    },
+    {
+      kind: StatefulSetModel.kind,
+      namespaced: true,
+      isList: true,
+      selector: { release: `${helmReleaseName}` },
+    },
+    {
+      kind: PodModel.kind,
+      namespaced: true,
+      isList: true,
+      selector: { release: `${helmReleaseName}` },
+    },
+    {
+      kind: SecretModel.kind,
+      namespaced: true,
+      isList: true,
+      selector: { name: `${helmReleaseName}` },
+    },
+  ];
```

Comment:
@debsmita1 @serenamarie125 how did we come up with this subset of resources? We cannot find EVERY resource that is in the helm release by using our standard client APIs.

Reply:
@christianvogt I saw @siamaksade 's comment here : https://docs.google.com/document/d/1mOYvOvcixmgfEJJhhGql9Wwzk3daZZHMtD5dQD29xSY/edit?disco=AAAAECMJpYY

Reply:
cc @siamaksade 
I'm assuming we want to see all resources in a helm release?

Reply:
@christianvogt yes, all resources in a helm release